heidegger_index/templates/markdown/_source.md

```diff
@@ -0,0 +1,3 @@
+Source:
+André van Delft and Johannes de Wit, '{% if type == 'work' %}{{ work.title }}', in _Heidegger index_ (n.d.) \<<https://heidegger.delve.nu{% url 'index:work-detail' work.slug %}>\>,
```

Comment:
Hier staan wat suggesties om de URL https://heidegger.delve.nu automatisch te kunnen genereren (handig voor als we van domein wisselen: https://stackoverflow.com/questions/2345708/how-can-i-get-the-full-absolute-url-with-domain-in-django